src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -95,8 +97,14 @@ public JedisClusterInfoCache(final GenericObjectPoolConfig<Jedis> poolConfig,
 
   public JedisClusterInfoCache(final GenericObjectPoolConfig<Jedis> poolConfig,
       final JedisClientConfig clientConfig) {
+    this(poolConfig, clientConfig, null);
+  }
+
+  public JedisClusterInfoCache(final GenericObjectPoolConfig<Jedis> poolConfig,
```

Comment:
Do you plan to provide an interface for setting `JedisClusterInfoCache` for`JedisCluster`?

Reply:
I'm not planning such idea.

But I won't mind to see a PR :)